### PR TITLE
Translate Korean

### PR DESCRIPTION
**Pull request by jeyraof**

### CODE REVIEW

**Comment by stefansundin** [comment]
Hi!

It looks great. There are a few untranslated strings though, was this on purpose?

Also, how do you want to be credited? Simply with "Jeyraof"?

Thank you, this is just in time for the new version.

File: localization/ko_KR/installer.nsh [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+LangString L10N_LANG ${LANG_KOREAN} ""
+
+LangString L10N_UPGRADE_TITLE     0 "이미 설치 되었습니다."
+LangString L10N_UPGRADE_SUBTITLE  0 "${APP_NAME}을 설치할 방법을 선택하세요."
+LangString L10N_UPGRADE_HEADER    0 "이 컴퓨터에 ${APP_NAME}이 이미 설치되어 있습니다. 원하는 동작을 선택하고, 다음을 눌러 진행하세요."
+LangString L10N_UPGRADE_UPGRADE   0 "(&U) ${APP_NAME}을 ${APP_VERSION}로 업그레이드 합니다."
+LangString L10N_UPGRADE_INI       0 "기존 세팅을 ${APP_NAME}-old.ini 파일로 백업합니다."
+LangString L10N_UPGRADE_INSTALL   0 "(&I) 새로운 위치에 설치합니다."
+LangString L10N_UPGRADE_UNINSTALL 0 "${APP_NAME}을 제거합니다."
+LangString L10N_UPDATE_DIALOG     0 "새로운 버젼이 있습니다.$\n설치를 중단하고 웹사이트를 열시겠습니까?"
```
Comment (devxoul) [inline localization/ko_KR/installer.nsh]: According to [The National Institute of the Korean Language](http://www.korean.go.kr/front/foreignSpell/foreignSpellView.do?forgn_seq=7477&mn_id=96), '버전' is the right orthography for 'version'.

**Comment by jeyraof** [comment]
I think, you say `AboutBox and contents` are not translated. Right?
I had appended it. please re-check.

And I like @jeyraof on credit message.

Thanks! :+1:

File: localization/ko_KR/installer.nsh (hunk 1 above)
Comment (jeyraof) [inline localization/ko_KR/installer.nsh]: :+1:

**Comment by stefansundin** [comment]
The only string I see that is left untranslated now is `Open &ini file`; https://github.com/stefansundin/altdrag/pull/37/files#diff-647fe4630d384ccc759afcb73d26567cR118

Otherwise, everything is good!

**Comment by jeyraof** [comment]
@stefansundin lol.. that was mistake.. :P. Thanks a lot!

**Comment by stefansundin** [comment]
Thank you!

**Comment by stefansundin** [comment]
I made a few small changes to your files: https://github.com/stefansundin/altdrag/commit/1110ffaf6509e27be57e02977378bb4b117d9e65

I forgot to mention that translating `strings.h` is not really necessary. I automatically generate that file from your `Translation.ini`. I totally forgot about this, sorry if you had to do extra work because of it.

I amended the translation documentation with a note, so that I don't forget about this again: https://stefansundin.github.io/altdrag/doc/translate.html

Also, the installer looked great, I don't know if you tested that.

**Comment by jeyraof** [comment]
Great! Cheers!

**Pull request merged**